### ISSUE TITLE
How to specify which shell to use?

### ISSUE THREAD

**Issue by saurabhnanda** [ndmitchell/shake#764]
I have a command which needs RVM's shell helpers to be sourced in the current shell. They're setup in `~/.bashrc` AND `~/.profile`.

I tried the following...

```
cmd_ Shell ["rvm", "use", "2.3.7", "&&", "bundle", "install"]
````

... which results in ...

```
* Raised the exception:
Development.Shake.cmd, system command failed
Command line: /bin/sh -c 'rvm use 2.3.7 && bundle install'
Original command line: rvm use 2.3.7 && bundle install
```

Is there any way to tell shake to use `bash` instead of `sh`?

**Comment by ndmitchell** [comment]
Shake always uses `sh`, or rather the underlying Shell process execution of the Haskell `process` library, which itself uses `sh`. But you can use "/bin/bash -c 'rvm use 2.3.7 && bundle install'" as the command, not specify `Shell` at all, and it should work.

**Comment by saurabhnanda** [comment]
Yes. I got around to using /bin/bash manually but I was hoping to specify
that as a top-level config. A very easy runtime failure to avoid especially
when a number of commands are dealing with rvm and nvm.

**Comment by ndmitchell** [comment]
Not unreasonable. A (ShellCommand String :: CmdOption) would be possible, so you could specify `ShellCommand "/bin/bash"`? My only reservation is that Shell works very differently on Windows and Linux, so if you specify ShellCommand then you're going to be losing that flexibility. In previous systems I've worked on I've defined:

```
bash x = cmd_ $ "/bin/bash -c '" ++ x ++ "'"
```

And that  has worked very well.
